### PR TITLE
Mpris bugfix

### PR DESCRIPTION
**Pull request by valdur55**
Intial scope:
Draft to fix issues which described in bugreport #2076
BrokenDbus supports now Play/Pause/Stop/Next/Prev media commands.
Player metadata and playbackStatus gets updated during BrokenDBusMpris init.
BrokenDbus PlayerOnChange subscription is disconnected during player_remove.
For easier debugging I made some exceptions more exceptional and reduced player_add and player_remove calls count. 

Current scope from commit 94e5650
Calculate ptime only when time is in format.
Use one callback to handle all players PlayerOnChange event.
PlayerOnChange event handler really knows signal sender.
self._player_details.get("id") - > self._player_details.get("_id")

### CODE REVIEW

File: py3status/modules/mpris.py [hunk 1]
```diff
@@ -256,15 +256,15 @@ def _get_text(self):
         ptime = None
 
         if hasattr(self._player, "Position") and self.py3.format_contains(
```
Comment (valdur55) [inline py3status/modules/mpris.py:258]: This line black formatting is ugly.

Comment (valdur55) [reply py3status/modules/mpris.py:258]: Force push broke things... This comment isn't at corret line.

**Comment by valdur55** [comment]
For now I have used it for a while. Time for conclusion.

**Key changes for mpris module:**
1. Using mpris2 and pydbus as backend which allows handle chrome/chromium browser as same way as other players. ( Probably best fix for #1827 )
2. Added config button_next_player and button_prev_player which switches between same status players as top priority player
3. Added config player_hide_non_canplay for hiding chrome/chromium mpris instances which otherwise hides only when chrome/chromium is closed after playing some media.
4. Added button_switch_to_top_player for switching top priority player.Player priority and name index handles corretnly vlc and chrome.instanceXXX.
5. Requesting for mediaplayer metadata only when it is really used.
   It is in two stages. 
       1. If no metadata used mpris will deny all metadata player_on_change calls. 
       2. If some metadata is used then asked metadata ammont is checked against format. (format_contains).
6. Subscribing one function for handling mediaplayer properties change and function knows signal sender player.
7. Sending play/pause/next/prev/stop signal to player will trigger update only once after mediaplayer has reported new state.

**Additional out of mpris scope changes:**
* events.py After i3wm reload first mouse event triggered unexpected error.
* core.py Drop negative update_due to prevent endless loop which causes high cpu. More info: #2085

**Comment by artizirk** [comment]
Tested this on my computer, seems to work well!

**Comment by ultrabug** [comment]
Thanks a lot for your hard work @valdur55

**Pull request merged**